### PR TITLE
feat: address follow-ups 

### PR DESCRIPTION
**Pull request by weboko**
#2

### CODE REVIEW

File: src/hooks/useWaku.ts
Comment (alrevuelta) [inline src/hooks/useWaku.ts:40]: is this tested? i get

```
2023-11-07 11:00:21 DBG 2023-11-07 10:00:21.868+00:00 Received error response from handler       tid=1 file=serverprivate.nim:269 status=400 meth=POST peer=*:43734 uri=/relay/v1/messages/%2Fwaku%2F2%2Fdefault-waku%2Fproto error_status="400 Bad Request" error_content_type=text/plain error_message=
```

Comment (weboko) [reply src/hooks/useWaku.ts:40]: not really, as per this discussion I need cors to be enabled 
https://discord.com/channels/1110799176264056863/1169610038491488267/1169649449669234738

**Comment by weboko** [comment]
@alrevuelta waiting for your approval

**Review by gabrielmer — APPROVED** [review]

**Pull request merged**
